### ISSUE TITLE
Issue with generated content not showing up correctly

### ISSUE THREAD

**Issue by gnickoloff** [sandywalker/webui-popover#128]
I am using the following code to generate a listing of related products with additional information opening up as a popover. What I am seeing is, although the source code is generated properly, the popover content is equal to the first instance on the page regardless of which link I click.

Here is the code I'm using to generate the list with "more information" popover links and content blocks, etc:
```php
<a class="popover" href="#pop_<?$i?>"><i class="icon-info"></i></a> 
<div id="pop_<?=$id?>" class="webui-popover-content">
    <img src="<?php if($aspect_ratio):?><?php echo $this->helper('catalog/image')
        ->init($_item, 'thumbnail')
        ->constrainOnly(FALSE)
        ->keepAspectRatio(TRUE)
        ->keepFrame(FALSE)
        ->resize($ratio_width);?>
    <?php else: ?><?php echo $this->helper('catalog/image')
            ->init($_item, 'thumbnail')
            ->resize($ratio_width,$ratio_height); ?>
    <?php endif; ?>" alt="<?php echo $this->escapeHtml($_item->getName()) ?>" />
	<br/>
	<p class=""><?php echo $_item->getSummaryDescription() ?></p>
</div>
```
```javascript    														
<script type="text/javascript">
//<![CDATA[
jQuery(document).ready(function($){ 
	$('a.popover').webuiPopover({ 
	        cache:false, 
		title:'<?=$this->escapeHtml($_item->getName())?>',
		trigger:'hover',
		animation:'pop',
		url:'#pop_<?=$id?>',
		width: 324,
		placement: 'top-left'
	}); 
});
//]]>
</script>
```
.... and here is the resulting HTML output: (notice that all three blocks are generated correctly and appear in the page source as expected)

FIRST INSTANCE:
```html
<a class="popover" href="#pop_6758"><i class="icon-info"></i></a> 
<div id="pop_6758" class="webui-popover-content">
       <img src="/media/catalog/product/cache/1/thumbnail/300x/17f82f742ffe127f42dca9de82fb58b1/1/1/1100-side.jpg" alt="Illuminator Arm Extreme">
        <br>
	<p class="">A tried and true workhorse that fits reflector panels up to 6 feet.</p>
</div>											
<script type="text/javascript">
//<![CDATA[
jQuery(document).ready(function($){ 
$('a.popover').webuiPopover({ 
	cache:false, 
	title:'Illuminator Arm Extreme',
	trigger:'hover',
	animation:'pop',
	url:'#pop_6758',
	width: 324,
	placement: 'top-left'
  }); 
});
//]]>
</script>
```
...//...
SECOND INSTANCE:
```html
<a class="popover" href="#pop_6853"><i class="icon-info"></i></a> 
<div id="pop_6853" class="webui-popover-content">
	<img src="/media/catalog/product/cache/1/thumbnail/300x/17f82f742ffe127f42dca9de82fb58b1/3/0/302-no-stand_1.jpg" alt="Basics Reflector Holder">
	<br>
	<p class="">A great cost-effective arm for beginners. </p>
</div>
<script type="text/javascript">
//<![CDATA[
jQuery(document).ready(function($){ 
  $('a.popover').webuiPopover({ 
	cache:false, 
	title:'Basics Reflector Holder',
	trigger:'hover',
	animation:'pop',
	url:'#pop_6853',
	width: 324,
	placement: 'top-left'
  }); 
});
//]]>
</script>
```
...//...
THIRD INSTANCE:
```html
<a class="popover" href="#pop_6977"><i class="icon-info"></i></a> 
<div id="pop_6977" class="webui-popover-content">
	<img src="/media/catalog/product/cache/1/thumbnail/300x/17f82f742ffe127f42dca9de82fb58b1/5/5/5571-dual_1.jpg" alt="Illuminator Arm Extreme">
	<br>
	<p class="">A perfect pair with the Omega Reflector, the extreme arm offers more non-traditional mounting options.  </p>
</div>
<script type="text/javascript">
//<![CDATA[
jQuery(document).ready(function($){ 
  $('a.popover').webuiPopover({ 
  	cache:false, 
	title:'Illuminator Arm Extreme',
	trigger:'hover',
	animation:'pop',
	url:'#pop_6977',
	width: 324,
	placement: 'top-left'
  }); 
});
//]]>
</script>
```

Yet, when any of the popover trigger, the content displayed is only the content on the first instance (regardless of which link was clicked to trigger the popover) even though the HTML in the page source shows the right content, etc. 

I've tried "cache:false" thinking the output for the popovers was being cached...but that does not good.

Another issue I'm having is that once one of the anchor links is clicked (or "hovered") it disappears from the page (page source) once the popover window closes making it impossible for someone to view it twice without reloading the page... is this the expected behavior? How can I disable that from happening?

**Comment by sandywalker** [comment]
The problem is that  the way you create popover will cause the conflicts.
There are two way to solve the problem.
First:
Attach an id to your a.popover element ,then init each popover by $('#id').webuiPopover({...}).
Second:
Just create all popovers once , do not create them again and again, instead, make the 'title' and 'url' option on your a.popover element like this
```html
<a class="popover" data-title="<?=$this->escapeHtml($_item->getName())?>" data-url="...." >...</a>
``` 
Then create popover only once
```js
jQuery(document).ready(function($){ 
  $('a.popover').webuiPopover({ 
    cache:false, 
    trigger:'hover',
    animation:'pop',
    width: 324,
    placement: 'top-left'
  }); 
});
```

**Comment by gnickoloff** [comment]
Excellent!  I used the first method. Working well now!
